relay-general/src/pii/processor.rs

```diff
@@ -59,6 +60,32 @@ impl<'a> Processor for PiiProcessor<'a> {
         meta: &mut Meta,
         state: &ProcessingState<'_>,
     ) -> ProcessingResult {
+        // Also apply pii scrubbing to the original value (set by normalization or other processors),
+        // such that we do not leak sensitive data through meta. Deletes `original_value` if an Error
+        // value is returned.
+
+        if let Some(Value::String(original_value)) = meta.original_value_as_mut() {
+            if let Some(parent) = state.iter().next() {
+                let attrs = FieldAttrs {
+                    pii: Pii::True,
+                    ..Default::default()
+                };
+
+                let new_state = parent.enter_static(
+                    "",
```

Comment:
should i put something in the key variable?

Reply:
Yes, the name of the current field (see suggestion below). Without that, name-based rules will not match.